.github/workflows/trunk.yml

```diff
@@ -160,36 +160,36 @@ jobs:
       AWS_OSSCI_METRICS_V2_ACCESS_KEY_ID: ${{ secrets.AWS_OSSCI_METRICS_V2_ACCESS_KEY_ID }}
       AWS_OSSCI_METRICS_V2_SECRET_ACCESS_KEY: ${{ secrets.AWS_OSSCI_METRICS_V2_SECRET_ACCESS_KEY }}
 
-  macos-10-15-py3-lite-interpreter-x86-64:
-    name: macos-10-15-py3-lite-interpreter-x86-64
+  macos-10-py3-x86-64-lite-interpreter-build:
+    name: macos-10-py3-x86-64-lite-interpreter-build
     uses: ./.github/workflows/_mac-build.yml
     with:
-      build-environment: macos-10-15-py3-lite-interpreter-x86-64
+      build-environment: macos-10-py3-lite-interpreter-x86-64
       xcode-version: "12"
       runner-type: macos-10.15
       build-generates-artifacts: false
     secrets:
       MACOS_SCCACHE_S3_ACCESS_KEY_ID: ${{ secrets.MACOS_SCCACHE_S3_ACCESS_KEY_ID }}
       MACOS_SCCACHE_S3_SECRET_ACCESS_KEY: ${{ secrets.MACOS_SCCACHE_S3_SECRET_ACCESS_KEY }}
 
-  macos-10-15-py3-arm64:
-    name: macos-10-15-py3-arm64
+  macos-12-py3-arm64-build:
+    name: macos-12-py3-arm64-build
     uses: ./.github/workflows/_mac-build.yml
     with:
-      build-environment: macos-10-15-py3-arm64
+      build-environment: macos-12-py3-arm64
       xcode-version: "13.3.1"
-      runner-type: macos-12
+      runner-type: macos-12-xl
       build-generates-artifacts: true
     secrets:
       MACOS_SCCACHE_S3_ACCESS_KEY_ID: ${{ secrets.MACOS_SCCACHE_S3_ACCESS_KEY_ID }}
       MACOS_SCCACHE_S3_SECRET_ACCESS_KEY: ${{ secrets.MACOS_SCCACHE_S3_SECRET_ACCESS_KEY }}
 
-  macos-12-3-py38-arm64-test:
-    name: macos-12.3-py3.8-arm64-test
+  macos-12-py3-arm64-mps-test:
+    name: macos-12-py3-arm64-mps-test
     uses: ./.github/workflows/_mac-test-arm64.yml
-    needs: macos-10-15-py3-arm64
+    needs: macos-12-py3-arm64-build
     with:
-      build-environment: macos-10-15-py3-arm64
+      build-environment: macos-12-py3-arm64
```

Comment:
For consistency with the other jobs here, do we want to specify the runner type here instead of `_mac-test-arm64.yml`?

Reply:
Perhaps leaving a comment here might suffice

Reply:
No need to do this here for sure.
I was checking the runners for all of these and realized this one is special!